src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -128,6 +133,15 @@ public override string ToString()
                 innerExceptionContents);
         }
 
+        /// <summary>
+        /// Returns if the the flow is retryable for the exception.
+        /// </summary>
+        /// <returns>A bool value to indicate if retry can be performed in case of the exception.</returns>
+        public bool isRetryable()
+        {
+            return Retry;
```

Comment:
You don't need an extra param `Retry` I think, You can just make this as a read-only property:

```
public bool IsRetryable
{
    get; 
}
```